packages/core/src/sass/app/_common/components/wallet.scss

```diff
@@ -1,63 +1,812 @@
+                    #151717,
+            ),
+        ),
     ),
     'ltc': (
-        'light': linear-gradient(to right, #dfe0e0, #e3e4e4, #e6e8e8, #eaecec, #eef0f0),
-        'dark': linear-gradient(to right, #4f5253, #575b5c, #5f6364, #676c6d, #6f7576),
+        'card': (
+            'light': (
+                'mobile':
+                    radial-gradient(
+                        100% 277.78% at 0% 100%,
+                        rgba(165, 168, 169, 0.24) 0%,
+                        rgba(193, 204, 207, 0.48) 100%
+                    )
+                    #ffffff,
+                'desktop':
+                    radial-gradient(
+                        100% 277.78% at 0% 100%,
+                        rgba(165, 168, 169, 0.24) 0%,
+                        rgba(193, 204, 207, 0.48) 100%
+                    )
+                    #ffffff,
+            ),
+            'dark': (
+                'mobile':
+                    radial-gradient(
+                        100% 277.78% at 0% 100%,
+                        rgba(165, 168, 169, 0.24) 0%,
+                        rgba(193, 204, 207, 0.48) 100%
+                    )
+                    #151717,
+                'desktop':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(165, 168, 169, 0.4) 0%,
+                        rgba(165, 168, 169, 0.16) 50.52%,
+                        rgba(193, 204, 207, 0.4) 100%
+                    )
+                    #ffffff,
+            ),
+        ),
+        'header': (
+            'light': (
+                'mobile':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(165, 168, 169, 0.4) 0%,
+                        rgba(165, 168, 169, 0.16) 50.52%,
+                        rgba(193, 204, 207, 0.4) 100%
+                    )
+                    #ffffff,
+                'desktop':
+                    radial-gradient(
+                        100% 277.78% at 0% 100%,
+                        rgba(165, 168, 169, 0.24) 0%,
+                        rgba(193, 204, 207, 0.48) 100%
+                    )
+                    #151717,
+            ),
+            'dark': (
+                'mobile':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(165, 168, 169, 0.4) 0%,
+                        rgba(165, 168, 169, 0.16) 50.52%,
+                        rgba(193, 204, 207, 0.4) 100%
+                    )
+                    #151717,
+                'desktop':
+                    radial-gradient(
+                        100% 4130.74% at 0% 100%,
+                        rgba(165, 168, 169, 0.4) 0%,
+                        rgba(165, 168, 169, 0.16) 50.52%,
+                        rgba(193, 204, 207, 0.4) 100%
+                    )
+                    #151717,
+            ),
+        ),
     ),
+    'usdt': $usdt-color,
```

Comment:
@hamid-deriv could we add `'ust': $usdt-color,` also because now when i select `Tether Omni` the currency in code is `ust` but we don't handle this currency here. as a result there is no bg color.
<img width="1778" alt="Screenshot 2023-06-16 at 15 46 26" src="https://github.com/binary-com/deriv-app/assets/103181646/02c1154e-4a48-4bf2-9e2d-d8c14d37aea0">
<img width="344" alt="Screenshot 2023-06-16 at 16 07 01" src="https://github.com/binary-com/deriv-app/assets/103181646/f0c30a04-7277-4bd5-aec0-787edf573027">
<img width="394" alt="Screenshot 2023-06-16 at 16 08 55" src="https://github.com/binary-com/deriv-app/assets/103181646/41bb6087-4aae-4e05-83c6-0de216a58d52">

Reply:
@heorhi-deriv Sure, I will add it. It wasn't on the list I checked. I missed it maybe 👍